core/pom.xml

```diff
@@ -570,8 +570,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.kohsuke.metainf-services</groupId>
       <artifactId>metainf-services</artifactId>
-      <version>1.4</version>
-      <scope>provided</scope>
```

Comment:
Any reason for the provided scope earlier? Maybe related to https://github.com/jenkinsci/core-annotation-processors?

Reply:
@dwnusbaum Core-annotation-processors can be patched -- it has stale dependencies too.  I have a fast patch for that too, but need to clean it up a tad.